src/baldr/edgeinfo.cc

```diff
@@ -445,6 +456,17 @@ json::MapPtr EdgeInfo::json() const {
     edge_info->emplace("speed_limit", static_cast<uint64_t>(speed_limit()));
   }
 
+  const auto cond_limits_range = GetTags().equal_range(TaggedValue::kConditionalSpeedLimits);
+  if (cond_limits_range.first != cond_limits_range.second) {
```

Comment:
Well, I doubt that I know a proper formatting for all tagged values, especially `kLandmark` and `BssInfo`. Probably worth to add in a separate PR

Reply:
we really just wanted a switch case with only this one filled out so that we have a template to fill out the others. you dont have to fill the others out just would be nice to have a switch here so we see there is work to do

Reply:
Added in dbe27c30b6cb2716a8bcd7223e1d539b3d3deb9f